.travis.yml

```diff
@@ -35,6 +35,11 @@ script:
     - flake8 . --max-line-length=120 --filename=pre-*.py,post-*.py
     - npm install -g less less-plugin-clean-css
     - pip install -q -r requirements.txt
+# don't use pypi's openupgradelib, but the one from source to avoid choking
+# on unreleased features in openupgradelib
+    - pip uninstall openupgradelib
+    - git clone https://github.com/OCA/openupgradelib ~/openupgradelib
+    - pip install ~/openupgradelib
```

Comment:
You can install directly with pip: http://stackoverflow.com/questions/20101834/pip-install-from-github-repo-branch